mlrun/datastore/sources.py

```diff
@@ -269,17 +275,20 @@ def to_step(
     ):
         import storey
 
+        # give priority to source time_field over the feature set's timestamp_key
+        if self.time_field:
+            time_field = self.time_field
+
         attributes = self.attributes or {}
         if context:
             attributes["context"] = context
         return storey.ParquetSource(
             paths=self.path,
             key_field=self.key_field or key_field,
-            time_field=self.time_field or time_field,
             storage_options=self._get_store().get_storage_options(),
             end_filter=self.end_time,
             start_filter=self.start_time,
-            filter_column=self.time_field or time_field,
```

Comment:
You could have just used the old code here, and remove the 4 line addition previously in this function (the code already gave priority to `self.time_field` over the provided `time_field`)